src/content/home/Home.module.css

```diff
@@ -0,0 +1,215 @@
+.home {
+  display: grid;
+  grid-template-rows: minmax(550px, auto) 85px;
+}
+
+.main {
+  display: grid;
+  grid-template-rows: auto 1fr;
+  background-image: linear-gradient(var(--color-black), var(--color-blue));
+  padding-top: 40px;
+}
+
+.mainTop {
+  position: relative;
+}
+
+.mainTopMiddle {
+  display: grid;
+  grid-template-rows: auto 110px auto;
+  justify-content: center;
+}
+
+.mainTopMiddle .logotype {
+  height: 36px;
+  fill: var(--color-white);
+  justify-self: center;
+}
+
+.mainTopMiddle .text {
+  text-align: center;
+  font-size: 18px;
+  font-weight: var(--color-light);
+  color: var(--color-white);
+  margin-top: 15px;
+}
+
+.appList {
+  display: grid;
+  grid-template-columns: repeat(3, 270px);
+}
+
+.appListItem {
+  display: grid;
+  grid-template-rows: 56px auto;
+  justify-content: center;
+}
+.appListItem .button {
+  display: block;
+  position: relative;
+  width: 221px;
+  height: 40px;
+  background-color: var(--color-blue);
+  color: var(--color-white);
+  border-radius: 6px;
+  text-align: center;
+  padding: 5px;
+  font-size: 18px;
+  font-weight: var(--color-bold);
+}
+
+.appListItem .button span {
+  margin-left: -20px;
+}
+
+.appListItem .button .appIconWrapper {
```

Comment:
When you unwrap the nested classes, check if that specificity was really needed. For example, would just the `.appIconWrapper` be sufficient?